internal/pkg/policy/policy_output.go

```diff
@@ -157,7 +157,7 @@ func (p *Output) prepareElasticsearch(
 			return err
 		}
 
-		if err = bulker.Update(ctx, dl.FleetAgents, agent.Id, body); err != nil {
+		if err = bulker.Update(ctx, dl.FleetAgents, agent.Id, body, bulk.WithRefresh(), bulk.WithRetryOnConflict(3)); err != nil {
```

Comment:
1. What does `WithRefresh` do?
2. Is there any place where we don't want these options? Can we make the default so we can't forget the next time someone adds an Update request?

Reply:
It is not standard elasticsearch for refresh to be there, and there are places where we do not want refresh. It just happens that in most of these cases we do want refresh.

Reply:
https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-refresh.html